src/core_modules/capture-core/components/WidgetTrackedEntityRelationship/WidgetTrackedEntityRelationship.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React, { useCallback, useState } from 'react';
+import { Button } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { NewTrackedEntityRelationship } from './NewTrackedEntityRelationship/NewTrackedEntityRelationship.container';
+import type { Props } from './WidgetTrackedEntityRelationship.types';
+
+export const WidgetTrackedEntityRelationship = ({ relationshipTypes, renderRef, trackedEntityType }: Props) => {
```

Comment:
Hey, 
Can you have a look at the `WidgetTrackedEntityRelationship` contents width? On a laptop screen, you can see other widgets below. It looks fine on bigger screens. 

<img width="1081" alt="Screenshot 2022-02-07 at 15 45 18" src="https://user-images.githubusercontent.com/6817073/152810860-b0ff653f-9379-407b-bfb1-8c6aef512103.png">

Thanks!